app/components/Form/SavingIndicator.tsx

```diff
@@ -6,15 +6,9 @@ interface Props {
   lastEdited: Date;
```

Comment:
you should remove this prop, and edit the places where this component is used. There's probably a fair amount of code that can be removed here